src/components/companies/MiddleBlock.js

```diff
@@ -24,13 +25,18 @@ const useStyles = makeStyles({
     fontWeight: '600',
     fontSize: fontSizing(6),
     lineHeight: '56px',
+    [theme.breakpoints.down('sm')]: { fontSize: fontSizing(4) },
+    [theme.breakpoints.down('xs')]: {
```

Comment:
Are we finding these same breakpoint styles on most of the mobile pages? Or are they different for almost every page?